lib/rops/index.js

```diff
@@ -3,6 +3,37 @@ const csv = require('csv-stringify');
 const AWS = require('aws-sdk');
 const archiver = require('archiver');
 
+const proceduresColumns = [
+  { key: 'ropId', header: 'return_id' },
+  { key: 'id', header: 'id' },
+  { key: 'species', header: 'species' },
+  { key: 'severityNum', header: 'no_of_procedures' },
+  { key: 'reuse', header: 'reuse' },
+  { key: 'placeOfBirth', header: 'place_of_birth' },
+  { key: 'nhpsOrigin', header: 'nhp_place_of_birth' },
+  { key: 'nhpsGeneration', header: 'nhp_generation' },
+  { key: 'ga', header: 'genetic_status' },
+  { key: 'new_genetic_line', header: 'creation_of_new_genetic_line' },
+  { key: 'purposes', header: 'purpose' },
+  { key: 'subPurpose', header: 'sub_purpose' },
```

Comment:
is this `subPurposes`? I know it *shouldn't* be, but the same field names were used for a rop and a proc, hence the plural

Reply:
ignore me, I continued  to read the codez